Automattic/wp-calypso
# Hide Unseen Count in Reader

#### Changes proposed in this Pull Request

* Add checks to the reader sidebar "Followed" tab to hide unseen count for non-eligible users.
* This uses the `isEligibleForUnseen` method to check if the user should be able to see the unseen count. 

#### Testing instructions

* Go to http://calypso.localhost:3000/read and login with an A8C account
* Click 'Following' tab and verify the counts are all visible
* Log out and log in again as a NON-A8C user
* Click 'Following' tab and verify the counts are *NOT* visible

#### Screenshot

**NON**
<img width="661" alt="Markup 2021-11-20 at 19 03 23" src="https://user-images.githubusercontent.com/33258733/142736909-21b07aa3-af61-4363-b6d9-a5452dbe4c52.png">

**A8C**
<img width="670" alt="Markup 2021-11-20 at 19 03 04" src="https://user-images.githubusercontent.com/33258733/142736939-a7fcbdcd-df66-4a89-926a-51f660d2ff80.png">


Related to #57015
